src/base/DataSpan.zig

```diff
@@ -0,0 +1,2 @@
+start: u32,
```

Comment:
I think this is too small to deserve a file, I'd put this in https://github.com/roc-lang/roc/blob/main/src/base.zig